app/com/linkedin/drelephant/exceptions/TonYExceptionFingerprinting.java

```diff
@@ -361,4 +368,56 @@ private boolean isExceptionLogSimilar(String exceptionStackTrace) {
     }
     return false;
   }
+
+
+  /**
+   * @param stackTrace String containing stackTrace of exception seperated by new line
+   * @param limit maximum number of lines required in stackTrace
+   * @return stackTrace with given number of lines or less
+   */
+  private String truncateStackTrace(String stackTrace, int limit) {
```

Comment:
This method is not used anywhere.

Reply:
don't know how this method added , it's not even related to EF classification . I'll remove it